external/llvm-project/mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -635,6 +635,9 @@ elementwiseMatchAndRewriteHelper(Operation *operation,
         newShape.push_back(it.value());
         affineExprs.push_back(
             mlir::getAffineDimExpr(it.index(), rewriter.getContext()));
+      } else if (it.value() == 1) {
```

Comment:
I'm not sure that sort of special-case fix should be happening? Didn't upstream point out a pass we could use that would clean some of this stuff up?